homeassistant/components/ryse/manifest.json

```
@@ -0,0 +1,13 @@
+{
+  "domain": "ryse",
+  "name": "RYSE",
+  "bluetooth": [],
+  "codeowners": ["@mohamedkalel82"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/ryse",
+  "integration_type": "device",
+  "iot_class": "local_push",
+  "quality_scale": "bronze",
+  "requirements": ["bleak==1.0.1", "ryseble==1.5.0"]
```

bleak is already a core dependency

```suggestion
  "requirements": ["ryseble==1.5.0"]
```